interop/run.py

```diff
@@ -46,22 +56,30 @@ def __init__(self):
         }
 
     # (re-) build the docker images
-    os.system("docker-compose -f ../docker-compose.yml -f interop.yml build sim" )
+    logging.info("Building sim image")
```

Comment:
I already had the code ready to remove this, but apparently didn't push it.

Reply:
I think we need to offer the option of building locally as well... it might be helpful for rapid iteration. I'll see if I can add this as an option.